build-tools/installers/create-installers.targets

```diff
@@ -270,7 +270,7 @@
     <_MSBuildFiles Include="$(MSBuildSrcDir)\Xamarin.Android.Bindings.JarToXml.targets" ExcludeFromAndroidNETSdk="true" />
     <_MSBuildFiles Include="$(MSBuildSrcDir)\Xamarin.Android.Build.Tasks.dll" />
     <_MSBuildFiles Include="$(MSBuildSrcDir)\Xamarin.Android.Build.Tasks.pdb" />
-    <_MSBuildFiles Include="@(_LocalizationLanguages->'$(MSBuildSrcDir)\%(Identity)\Java.Interop.Localization.resources.dll')" />
+    <_MSBuildFiles Include="@(_LocalizationLanguages->'$(MSBuildSrcDir)\%(Identity)\Java.Interop.Localization.resources.dll')" ExcludeFromAndroidNETSdk="true" />
```

Comment:
Does this file not exist now?

Reply:
I think this change is not strictly necessary, but I figured it would be best to pull these `Java.Interop.Localization.resources.dll` files from the same location that we take `Java.Interop.Localization.dll` and other JI tools from for the .NET SDK (e.g. https://github.com/xamarin/xamarin-android/pull/7028/files#diff-0134c03744a69c35881982fb2140cf780816fe186f0b6fac66477a8416d2fca6R82)